homeassistant/components/dunehd/media_player.py

```diff
@@ -95,22 +114,22 @@ def state(self):
         return state
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the device."""
         return self._name
 
     @property
-    def available(self):
+    def available(self) -> bool:
         """Return True if entity is available."""
-        return bool(self._state)
+        return len(self._state) > 0
```

Comment:
Why did we change this?

Reply:
I think this is a bit more explicit. "converting _something_ to bool" vs "checking that some _collection_ is not empty".